pkg/api/routers/metric.go

```diff
@@ -16,6 +16,7 @@ package routers
 
 import (
 	"github.com/astaxie/beego"
+
```

Comment:
Please rename this file to pkg/api/routers/metrics.go

Reply:
done